models/model_factory.py

```diff
@@ -1,12 +1,12 @@
 import torch
 
-from .classification import EfficientNet
+from .classification import EfficientNetWrapper
 from .gans import WGAN
 
 class ModelFactory:
     def create(model_name: str) -> torch.nn.Module:
         if model_name == 'efficientnet':
-            model = EfficientNet()
+            model = EfficientNetWrapper('efficientnet-b7')
```

Comment:
Actually I think if we use 'efficientnet-l2' the weights aren't pre-trained

Reply:
We can use from_name to get untrained version